test_common/page_objects/feedback_page.rb

```diff
@@ -0,0 +1,21 @@
+module ET3
+  module Test
+    class FeedbackPage < BasePage
+      set_url '/feedback'
+
+      def fill_in_feedback(feedback)
+        fill_in 'Have you had any problems using this service?',  with: feedback.problems
```

Comment:
This cannot work for welsh ?  Please use something similar to this from the claimants_details_page, changing names and I18n stuff accordingly

```ruby
section :claimants_name_question, :question_labelled, 'questions.claimants_name.label', exact: false do
        element :field, :css, "input"
        element :error_contains_numbers, :exact_error_text, 'errors.custom.claimants_name.not_a_number', exact: false
        element :error_invalid_name, :exact_error_text, 'errors.messages.invalid_name', exact: false
         def set(*args); field.set(*args); end
      end
```

Reply:
The error elements are there just to allow your test to check for errors which of course are also in different languages